lib/merchant_repository.rb

```diff
@@ -1,3 +1,4 @@
+require 'pry'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.